### DISCUSSION TITLE
Cloudinary image adding path to URL (/f_auto,c_fit,h_773,w_1400/)

### DISCUSSION THREAD

**Discussion by 0xtiago** [gethinode/hinode#1049] [Help] [answered]
Hi guys, 

I've tried to embed some gifs hosted in cloudinary in my blog posts, and JUST WITH GIFs, but im facing some problems.
The line bellow is an example of image:

`{{< image src="https://res.cloudinary.com/dtr6hzxnx/image/upload/v1721593055/blog/horusec_csharp.gif" caption="">}}`

When the page is rendered by the browser, it automatically adds in the middle `/f_auto,c_fit,h_773,w_1400/`...
https://res.cloudinary.com/dtr6hzxnx/image/upload/f_auto,c_fit,h_773,w_1400/v1721593055/blog/horusec_csharp.gif

![image](https://github.com/user-attachments/assets/e95220bc-f642-410c-b749-f402d2679419)

... and the image cannot be found.

My server img-src is also configured, and now I dont know what to do... :( 

![image](https://github.com/user-attachments/assets/0f776cf5-41f3-4476-9161-5f12503bcaff)

Have anybody went through the same problem?

Thanks!

**Comment by markdumay** [comment, marked as answer]
By default, Hinode generates [image sets](https://gethinode.com/docs/components/image/#arguments) to reduce the download size of an image for smaller screens. You can specify `plain=true` to include the original, unprocessed image instead. Your example should then work correctly.

I did some additional investigation, and it seems your example image is rather large. I tested the following shortcode locally:

```markdown
{{< image src="https://res.cloudinary.com/dtr6hzxnx/image/upload/v1721593055/blog/horusec_csharp.gif" caption="" plain=false >}}
```

Chrome gives me the following network error (Debugging tools, click on the filename `horusec_csharp.gif` in the Network tab and verify the response headers):

```bash
X-Cld-Error: Maximum total number of pixels in all frames/pages is 50 Megapixels. Requested 163.89 Megapixels
```

Cloudinary recommends to [convert animated gifs](https://cloudinary.com/documentation/animated_images#converting_an_animated_gif_to_video) to either `webm` or `mp4`. Unfortunately, videos are not supported by Hinode's `image` shortcode. If you're interested, you could [submit a feature request](https://github.com/gethinode/hinode/issues/new/choose) for a new shortcode, e.g. `video`, that supports both local and remote videos.

**Reply by 0xtiago** [reply]
Hi Mark, I try to convert my gif to a smaller size. I reply back in a couple of hours...thank yoU!
